src/matchers.rs

```diff
@@ -207,6 +207,10 @@ impl PathExactMatcher {
     pub fn new<T: Into<String>>(path: T) -> Self {
         let path = path.into();
 
+        if path.contains('?') {
+            panic!("Wiremock can't match the path `{}` because it contains a `?`. Use `wiremock::matchers::query_param` instead.", path)
```

Comment:
```suggestion
            panic!("Wiremock can't match the path `{}` because it contains a `?`. You must use `wiremock::matchers::query_param` to match on query parameters (the part of the path after the `?`).", path)
```